.travis.yml

```diff
@@ -1,8 +1,15 @@
 sudo: false
 language: node_js
 node_js:
-  - "0.10"
-  - "0.12"
-  - "iojs"
+  - stable
```

Comment:
`stable` is deprecated. https://github.com/travis-ci/docs-travis-ci-com/pull/534